db/model/collector.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/collector.js
+ */
+
+const constants = require('../constants');
+const assoc = {};
+
+module.exports = function collector(seq, dataTypes) {
+  const Collector = seq.define('Collector', {
+    id: {
+      type: dataTypes.UUID,
+      primaryKey: true,
+      defaultValue: dataTypes.UUIDV4,
+    },
+    name: {
+      type: dataTypes.STRING(constants.fieldlen.normalName),
+      allowNull: false,
+      validate: {
+        is: constants.nameRegex,
+      },
+    },
+    description: {
+      type: dataTypes.STRING(constants.fieldlen.longish),
+    },
+    helpEmail: {
+      type: dataTypes.STRING(constants.fieldlen.email),
+      validate: { isEmail: true },
+    },
+    helpUrl: {
+      type: dataTypes.STRING(constants.fieldlen.url),
+      validate: { isUrl: true },
+    },
+    registered: {
+      type: dataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: true,
```

Comment:
This record will only ever be created by a collector registering itself, so registered should default to true.